Install perfsonar binaries and libraries into their Debian proper location.
--- a/Makefile
+++ b/Makefile
@@ -3,10 +3,10 @@
 CONFIGPATH=/etc/perfsonar
 VERSION=3.5.1
 RELEASE=0.0.a1
-BIN=$(DESTDIR)/bin
-DOC=$(DESTDIR)/doc
-ETC=$(DESTDIR)/etc
-LIB=$(DESTDIR)/lib
+BIN=$(DESTDIR)/usr/lib/perfsonar
+DOC=$(DESTDIR)/usr/share/doc/perfsonar
+ETC=$(DESTDIR)/etc/perfsonar
+LIB=$(DESTDIR)/usr/lib/perfsonar
 
 default:
 	@echo No need to build the package. Just run \"make install\"
